### ISSUE TITLE
Do not allow more places to be selected within the same design. 

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#2577]
When a place was selected (final designs), do not allow more places to be selected within the same design. To reproduce: make sure you have 5 designs, select paces from 1 to 3, select an additional design, remove first place

![screen shot 2018-10-03 at 4 31 37 pm](https://user-images.githubusercontent.com/11248478/46406637-d3744880-c729-11e8-8e6d-6d66a7f1b60c.png)

ref - https://github.com/appirio-tech/connect-app/issues/2301

**Comment by sumitdaga** [comment]
@maxceem 
please assign this to me!

**Comment by sumitdaga** [comment]
@maxceem 
can you please share a sample project link , if you have it, where i can test this easily?

**Comment by sumitdaga** [comment]
@maxceem 
thanks!
but i can't figure what exactly the issue is ..we can't select more than 3 places ...thats fine!
what's the issue?

**Comment by maxceem** [comment]
@sumitdaga when we select some place for one design, for example, `1` we shouldn't be able to press another place for the same design, like `2` or `3`:
![image](https://user-images.githubusercontent.com/146016/46720125-2f971980-cca2-11e8-88b5-bf333be52e41.png)
- `2` and `3` has to become disabled in this case.

To select `2` or `3` for this design, we have `unselect` `1` first, and after we have to be able to choose another vacant place:
![image](https://user-images.githubusercontent.com/146016/46720186-62411200-cca2-11e8-84c6-99cf8f67bd24.png)

**Comment by sumitdaga** [comment]
@maxceem 
btw i found another issue ...after selecting a position say `1` ... `1` on other designs becomes visually inactive but still can be selected 
should i fix this as part of this issue?

**Comment by maxceem** [comment]
@sumitdaga yes, please. I'm slightly increasing prize for this issue to $40 for the additional case.
Thanks.

**Comment by sumitdaga** [comment]
@maxceem 
ready for review
PR #2595 
i would be happier with $45 or 50 ...
the logic to this was not straightforward

**Comment by maxceem** [comment]
@sumitdaga I've reviewed the PR, the prize is increased to $50 for taking care about the additional case. Thanks.
